DOCKER.md

```diff
@@ -117,6 +148,13 @@ ran after the `docker-compose up` process has been stopped. All relevant
 containers and images can then be optionally discarded using the `docker` CLI
 commands described in the previous paragraph.
 
+Use `docker ps -a` to show a list of all containers. Pass a container id to
+`docker rm` (ex: `docker rm 5c15ee2f1c4f`) to remove it.
+
+Use `docker images` to show a list of all images. Typically, you'll want to
+keep these images if you plan on running Docker with them again in the future.
+If you are done with them, you can pass an image id to `docker rmi` to remove
+an image (ex: `docker rmi 4253856b2570`).
```

Comment:
As an aside, I'm so thrilled with this documentation. This is something I wish I had as an early career developer interested in making contributions to open source projects, but still learning the basics with Docker. Thanks for taking the time to break this down!

Reply:
My pleasure! I was delighted to see this!